src/components/Notes/Notes.jsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  HStack,
+  SimpleGrid,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+
+import { updateCardById } from "../../actions/Card";
+import useUser from "../../lib/hooks/useUser";
+import AddNewNote from "./AddNewNote";
+import Note from "./Note";
+import { SentimentButton } from "./utils";
+
+const Notes = ({ cardId, notes, setCards }) => {
+  const [revNotes, setRevNotes] = useState(notes.map((n) => n).reverse());
```

Comment:
please call this currentNotes or something similar. i got a bit confused on what revNotes meant